config/local_docker/.genesis.env

```diff
@@ -3,7 +3,6 @@ GENESIS_PATH=./genesis.json
 BASE_ROLLUP_CFG_PATH=./base_sp_rollup.json
 ROLLUP_CFG_PATH=./sp_rollup.json
 GENESIS_EXPORTED_HASH_PATH=./genesis_hash.json
-
 L1_NETWORK=localhost
```

Comment:
We're missing a line here, defining `L1_PERIOD`, something I moved out of the scripts. Unless the scripts go reverted, this won't work. Insert/use `L1_PERIOD=12`.